lib/process-services-cloud/src/lib/form/components/widgets/date/date-cloud.widget.ts

```diff
@@ -53,74 +78,138 @@ export class DateCloudWidgetComponent extends WidgetComponent implements OnInit,
     maxDate: Date = null;
     startAt: Date = null;
 
-    /**
-     * Current date value.
-     * The value is always stored in the format `dd-MM-yyyy`,
-     * but displayed in the UI component using `dateDisplayFormat`
-     */
-    @Input()
-    value: any = null;
+    dateInputControl: FormControl<Date>;
 
-    private onDestroy$ = new Subject<boolean>();
+    private dateChangesSubscription: Subscription;
```

Comment:
Please use onDestroy$ subject rather than Subscription. Also, the onDestroy$ subject can be of type void